### PR TITLE
Add CryptoPanic

### PR DESCRIPTION
**Pull request by soodaayush**

### CODE REVIEW

**Comment by Carlgo11** [comment]
Could you explain a bit what the site is about and what an account on the site gives access to?

**Comment by soodaayush** [comment]
CryptoPanic is a news aggregation website for news pertaining to cryptocurrency. The account allows you to follow certain news sources, allows you to post comments on news stories, and finally allows you to set up price alerts for cryptocurrencies. They seem to also have a subscription service, called PRO, in which their main features is that you can set up price alerts that are instant, and add any custom RSS/Atom feeds, Reddit and Twitter sources.

File: img/c/cryptopanic.com.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+        </g>
+    </g>
+</svg>
```
Comment (Carlgo11) [inline img/c/cryptopanic.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1024 1024"><path fill="#FD9941" d="M421.3 1016a510.3 510.3 0 0 0 444.3-133.7c-76.4-12.6-179.4-30.9-179.5-38.4-.2-10.2 211.8-43.6 253.4-50.1a512 512 0 0 0 17.8-28.8c-75-12.5-168.2-29.4-168.3-36.6-.2-8 128-30 204.7-42.3a512 512 0 0 0-835.4-544.2C235 154.6 340 173.3 340 180.4c-.2 11.1-221.6 45.3-256 50.6a512 512 0 0 0-17.3 28.2C142 271.8 237.2 289.1 237 296c-.1 8.5-130.9 30.7-207 42.9A511 511 0 0 0 0 512c0 62 11 121.6 31.3 176.6 15.2-7 32.1-17.4 50.2-27.5-33.6-4-51.3-11-51.3-11C81.7 638.4 167.3 606 167.3 606c4.3-3 8.7-6.1 12.7-7.3 34.4-22.5 63-44 85.6-64.8 28.3 1.2 65-1.8 97.9-26.5-22.3 7.5-75 4-75 4-51.6-9.2-74.7-39.9-68.5-42.6a1390.4 1390.4 0 0 0 130.5-60.6c13.5-22.4 66.9-98.6 133.7-100.1a81 81 0 0 0 55.3-22.6l95.4-87.8s52.8 13.2 54.7 78.2c2.2 63.1-45.9 98.4-39.3 114.8 7 14.5 77-26.4 77-26.4s15.7 61.7-14.7 83c-30.4 21.2-99.1 130.8-117.5 213.2-4.5 25.8-10.3 47.6-16 69.4.8 17.3-4.3 100-34 138.6l-5-25.6A605.3 605.3 0 0 1 444 989.4l-22.6 26.6Zm51-640 12.4-37.8 22.6-9.3c-9.8 55.3-35 47.1-35 47.1Z"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Thanks for the info!
By your description, I think the `cryptocurrencies` category would be a better fit.

**Review by Carlgo11 — APPROVED** [review]
Thank you! 🙂
Your contribution will be visible in around 10 minutes. Sometimes it takes longer due to server and browser cache.

**Pull request merged**